packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.test.tsx

```diff
@@ -36,18 +49,20 @@ describe('Dob', () => {
     expect(screen.getByRole('textbox', { name: /date of birth/i })).toBeInTheDocument();
   });
 
-  it('typing in the date picker input sets the date of birth', async () => {
-    const user = userEvent.setup();
+  // TODO : Fix this test case.
```

Comment:
Instead of commenting this out, just use `xit()` in place of `it()`

Reply:
Sorry Ian, I had replaced it with `it.skip` before seeing this comment, would that suffice, or if `xit` preferred?

Reply:
Oh! That's also fine to... I thought jest didn't support `.skip` with `it` but I guess I'm wrong!